demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -0,0 +1,76 @@
+import subprocess
+import os
+from demisto_sdk.commands.common.tools import print_error, print_warning
+
+NODE_MODULES_DIRECTORY = os.path.dirname(os.path.dirname(
```

Comment:
if used only in `are_modules_installed_for_verify` , why do we need it as global? please move to the function